feature_campaigns/src/test/java/cm/aptoide/pt/feature_campaigns/CampaignImplTest.kt

```diff
@@ -18,29 +18,28 @@ import org.junit.jupiter.params.provider.MethodSource
 @ExperimentalCoroutinesApi
 internal class CampaignImplTest {
```

Comment:
Now there is no external "normaliser" and injection happens inside...
Mb we should also test the replace and append logic also? 🤔 
But I would say this class is pretty simple to spend time for doing this except to exercise in writing tests :-)

Reply:
Will leave this for the future, after we conclude all the scope tickets.